packages/icons/src/models/censhare-result-page.model.ts

```diff
@@ -101,6 +101,7 @@ export enum Businessfield {
   Kommunikation = 'kommunikation',
 }
 
+/* eslint-disable-next-line @typescript-eslint/no-empty-interface */
 export interface DetectedColorClass {}
```

Comment:
I think wherever `DetectedColorClass` is being used, it should be changed to `unknown` or `any`, because an emtpy interface is euqally helpful.

Reply:
I was able to replace it by `DetectedColor` type.